SpatialGDK/Extras/schema/spatial_debugging.schema

```diff
@@ -11,13 +11,6 @@ component SpatialDebugging {
      // The color for the authoritative virtual worker.
      uint32 authoritative_color = 2;
 
-     // Id assigned to the Unreal server worker which should be authoritative for this entity.
-     // 0 is reserved as an invalid/unset value.
-     uint32 intent_virtual_worker_id = 3;
-
-     // The color for the intended virtual worker.
-     uint32 intent_color = 4;
```

Comment:
Getting rid of intent debugging, intent is an implementation detail we should move away from exposing